pkg/apis/certmanager/v1alpha1/doc.go

```diff
@@ -0,0 +1,8 @@
+// +k8s:deepcopy-gen=package,register
+// +k8s:conversion-gen=github.com/munnerz/cert-manager/pkg/apis/certmanager
+// +k8s:openapi-gen=true
+// +k8s:defaulter-gen=TypeMeta
+
+// Package v1alpha1 is the v1alpha1 version of the API.
+// +groupName=certmanager.k8s.io
```

Comment:
I've gone with this group name, however I'm open to suggestions on alternatives!